api/v1alpha1/bundle_types.go

```diff
@@ -48,11 +48,17 @@ type BundleSpec struct {
 
 type BundleSource struct {
 	// Type defines the kind of Bundle content being sourced.
-	Type string `json:"type"`
+	Type BundleSourceType `json:"type"`
 	// Image is the bundle image that backs the content of this bundle.
 	Image *ImageSource `json:"image,omitempty"`
 }
 
+type BundleSourceType string
+
+const (
+	BundleSourceTypeImage BundleSourceType = "image"
+)
```

Comment:
Nice - these changes seem reasonable outside of the scope of this PR.

Reply:
@joelanford Following back up on this - is it worth splitting this change out into it's own PR while we continue to have reviewers parse through this and finalize on an implementation?